cruise_company_spring/src/main/java/com/example/cruise_company/service/impl/ReceiptServiceImpl.java

```diff
@@ -0,0 +1,83 @@
+package com.example.cruise_company.service.impl;
+
+import com.example.cruise_company.controller.dto.ReceiptDto;
+import com.example.cruise_company.service.ReceiptService;
+import com.example.cruise_company.service.model.Receipt;
+import com.example.cruise_company.service.repository.LinerRepository;
+import com.example.cruise_company.service.repository.ReceiptRepository;
+import com.example.cruise_company.service.repository.ReceiptStatusRepository;
+import com.example.cruise_company.service.repository.UserRepository;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component
```

Comment:
Use @Service annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers